src/codegeneration/BlockBindingTransformer.js

```diff
@@ -433,12 +433,14 @@ export class BlockBindingTransformer extends ParseTreeTransformer {
 
   transformBlock(tree) {
     var scope = this.pushScope(tree);
+    var prependBlockStatement = this.prependBlockStatement_;
+    this.prependBlockStatement_ = [];
```

Comment:
I think the meaning of this code would be easier to grasp quickly if the temp `var` was named eg `outerPrepends` to explain the role of the temp vs the member.